src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/SamlAssertion.cs

```diff
@@ -0,0 +1,671 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.IdentityModel;
+using System.IdentityModel.Claims;
+using System.IdentityModel.Policy;
+using System.IdentityModel.Selectors;
+using System.Runtime;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Security.Cryptography;
+using System.Xml;
+
+namespace System.IdentityModel.Tokens
+{
+    public class SamlAssertion // : ICanonicalWriterEndRootElementCallback  // Issue #31 in progress
```

Comment:
Thanks Matt

---
In reply to: [78483831](https://github.com/dotnet/wcf/pull/1515#discussion_r78483831) [](ancestors = 78483831)